src/main/java/no/unit/marc/Marc21XmlParser.java

```diff
@@ -28,33 +28,49 @@
 import java.util.Optional;
 import java.util.function.Consumer;
 
-public class RecordParser {
+public class Marc21XmlParser {
 
-    public static final String EMPTY_STRING = "";
-    public static final String CLOSING_BRACKET = ")";
+    private static final String EMPTY_STRING = "";
+    private static final String RECORD = "record";
+    private static final String DOES_NOT_START_WITH_RECORD_ERROR = "The xml string does "
+            + "not start with the <record> tag, please make sure that is does end and start with <record> </record>";
+    private static final String EXCEPTION_MESSAGE = "Caught an error while converting to "
+            + "reference object, make sure that the xml string used is  correct";
 
     /**
-     * Parses a SRU-response to extract the title of an marc21xml-record.
+     * Parses a SRU-response to convert it into a Reference object.
      *
      * @param xml marc21-xml record
      * @return simple json with <code>title</code>
```

Comment:
feil i dok?

Reply:
Jeg mente det var feil i den gamle ja, den henter jo ikke bare en tittel. Den konverterer jo hele inputen til et Reference-objekt

Reply:
men det er mulig @return skulle vært endret til simple json in the shape of a Reference-object

Reply:
@return er vel Reference objekt?

Reply:
Det er sant